src/app/loginComponents/accounts/external/accounts.component.html

```diff
@@ -17,54 +17,74 @@
+        </mat-card-content>
         <div>
-          Use with our CLI utilities or with our
-          <a [href]="dsServerURI + '/static/swagger-ui/index.html#'" target="_blank">REST web service interface</a> to work with and launch
-          tools and workflows. See the <a routerLink="/onboarding">Onboarding</a> page for more details on using the CLI.
+          <mat-form-field appearance="outline" class="w-75">
+            <mat-label class="accent-1-dark">
+              <strong>Token</strong>
+            </mat-label>
+            <input matInput readonly [type]="show ? 'text' : 'password'" value="{{ dockstoreToken }}" />
+            <button
+              mat-icon-button
+              matSuffix
+              class="accent-1-dark"
+              [cdkCopyToClipboard]="dockstoreToken"
+              (cbOnSuccess)="isCopied1 = true"
+              matTooltip="Copy Token"
+              appSnackbar
+            >
+              <mat-icon>file_copy</mat-icon>
+            </button>
+          </mat-form-field>
+          <button
+            [matTooltip]="show ? 'Hide token' : 'Show token'"
+            mat-button
+            color="secondary"
+            class="accent-1-dark small-mat-btn-skin small-btn-structure mat-elevation-z"
+            (click)="show = !show"
+          >
+            <mat-icon class="mat-icon-sm">{{ show ? 'visibility_off' : 'visibility' }}</mat-icon>
+            <span> Show </span>
+          </button>
         </div>
-      </mat-card-content>
-      <mat-form-field appearance="outline" class="w-75">
-        <mat-label class="accent-1-dark">
-          <strong>Token</strong>
-        </mat-label>
-        <input matInput readonly [type]="show ? 'text' : 'password'" value="{{ dockstoreToken }}" />
-        <button
-          mat-icon-button
-          matSuffix
-          class="accent-1-dark"
-          [cdkCopyToClipboard]="dockstoreToken"
-          (cbOnSuccess)="isCopied1 = true"
-          matTooltip="Copy Token"
-          appSnackbar
-        >
-          <mat-icon>file_copy</mat-icon>
-        </button>
-      </mat-form-field>
-      <button
-        [matTooltip]="show ? 'Hide token' : 'Show token'"
-        mat-button
-        color="secondary"
-        class="accent-1-dark small-mat-btn-skin small-btn-structure mat-elevation-z"
-        (click)="show = !show"
-      >
-        <mat-icon class="mat-icon-sm">{{ show ? 'visibility_off' : 'visibility' }}</mat-icon>
-        <span> Show </span>
-      </button>
+        <div>
+          <hr />
+          <span class="pull-right" fxLayoutAlign="end end">
+            <!-- This button needs to be vertically centered -->
+            <button
+              mat-button
+              class="accent-1-dark-btn small-mat-btn-skin small-btn-structure mt-2"
+              matTooltip="Revoke Dockstore token"
+              data-cy="revoke-token-button"
+              (click)="revokeToken()"
+            >
+              <mat-icon class="mat-icon-sm">link</mat-icon>
```

Comment:
I think we should use the `unlink` icon instead because the button revokes the token